### ISSUE TITLE
Accessibility: Asterisk on black tabs should be white

### ISSUE THREAD

**Issue by astridu** [sul-dlss/hungry-hungry-hippo#2042]
The asterisk should be white.

<img width="263" height="44" alt="Image" src="https://github.com/user-attachments/assets/c83d1719-bd2c-4a51-86cd-42101cb61ed3" />
